src/app/pages/register/register.component.ts

```diff
@@ -0,0 +1,206 @@
+import { Component, inject } from '@angular/core';
+import { Router, ActivatedRoute, RouterLink } from '@angular/router';
+import {
+  FormBuilder,
+  FormGroup,
+  Validators,
+  ReactiveFormsModule,
+  ValidationErrors,
+  FormControl,
+} from '@angular/forms';
+import { CommonModule } from '@angular/common';
+
+interface BundleService {
+  id: string;
+  name: string;
+  termsUrl: string;
+}
+
+interface Bundle {
+  id: string;
+  name: string;
+  services: BundleService[];
+}
+
+@Component({
+  selector: 'app-register',
+  templateUrl: './register.component.html',
+  imports: [RouterLink, ReactiveFormsModule, CommonModule],
+  styleUrl: './register.component.css',
+})
+export class RegisterComponent {
+  private router = inject(Router);
+  private route = inject(ActivatedRoute);
+  private formBuilder = inject(FormBuilder);
+
+  currentStep = 1;
+  totalSteps = 5;
+
+  bundles: Bundle[] = [
+    {
+      id: 'data-portal-galaxy',
+      name: 'Data Portal and Galaxy',
+      services: [
+        {
+          id: 'bpa',
+          name: 'Bioplatforms Australia Data Portal Terms and Conditions',
+          termsUrl: 'https://data.bioplatforms.com/',
+        },
+        {
+          id: 'galaxy',
+          name: 'Galaxy Australia Terms of Service',
+          termsUrl: 'https://site.usegalaxy.org.au/about#terms-of-service',
+        },
+      ],
+    },
+    {
+      id: 'tsi',
+      name: 'Threatened Species Initiative',
+      services: [
+        {
+          id: 'tsi',
+          name: 'TSI Terms and Conditions',
+          termsUrl: 'https://threatenedspeciesinitiative.com/',
+        },
+        {
+          id: 'bpa',
+          name: 'Bioplatforms Australia Data Portal Terms and Conditions',
+          termsUrl: 'https://data.bioplatforms.com/',
+        },
+        {
+          id: 'galaxy',
+          name: 'Galaxy Australia Terms of Service',
+          termsUrl: 'https://site.usegalaxy.org.au/about#terms-of-service',
+        },
+      ],
+    },
+  ];
+
+  bundleForm: FormGroup = this.formBuilder.group({
+    selectedBundle: ['', Validators.required],
+  });
+
+  // Password validator to require at least 8 characters including a lower-case letter, an upper-case letter, and a number
```

Comment:
The requirements from the current BPA stories say we also need a special character. Based on Auth0's settings I think the special characters should be `!@#$%^&*`